src/utils/hotcode-agent/src/main/java/io/bellsoft/hotcode/profiling/jfr/ExecutionSampleCounter.java

```diff
@@ -14,10 +14,6 @@ public class ExecutionSampleCounter implements Consumer<RecordedEvent> {
     private final Profile<Method> profile;
     private final int maxStackDepth;
 
-    public ExecutionSampleCounter(Profile<Method> topKStrategy) {
-        this(topKStrategy, -1);
-    }
-
     public ExecutionSampleCounter(Profile<Method> profile, int maxStackDepth) {
         this.profile = profile;
         this.maxStackDepth = maxStackDepth;
```

Comment:
This affects only the number of _processed_ frames, but not the number of _collected_ frames, right?